packages/grafana-data/package.json

```diff
@@ -33,6 +34,7 @@
     "rollup-plugin-terser": "4.0.4",
     "rollup-plugin-typescript2": "0.19.3",
     "rollup-plugin-visualizer": "0.9.2",
+    "sinon": "1.17.6",
```

Comment:
do not think we need this, is only one test that use it I think, should use jest.fn() instead

Reply:
I agree... but it is not a trivial change.  The test is using sinon for: `SinonFakeTimers`,  `clock = sinon.useFakeTimers(unix);`

Jest has `jest.useFakeTimers();` this does all the same things for interval, but does not let you reset the global clock.  Reading the jest docs on it I found https://github.com/taitulism/jest-fake-timers-issue-demo.

This test could be refactored to only use jest, but will require understanding the test a bit more and should not hold up this PR.